### PR TITLE
Np update subread dockers pd 2284

### PR DESCRIPTION
**Pull request by nikellepetrillo**
### Description

Updated a docker image in the festurecounts wdl as the old dockerfile was no longer buildable 

----

### Checklist 
If you can answer "yes" to the following items, please add a checkmark next to the appropriate checklist item(s) **and** notify our WARP documentation team by tagging either @ekiernan or @kayleemathews in a comment on this PR.

- [ ] Did you add inputs, outputs, or tasks to a workflow?
- [ ] Did you modify, delete or move: file paths, file names, input names, output names, or task names?
- [ ] If you made a changelog update, did you update the pipeline version number?

### CODE REVIEW

File: pipelines/skylab/multiome/Multiome.wdl [hunk 1]
```diff
@@ -18,7 +18,7 @@ workflow Multiome {
         File annotations_gtf
         File ref_genome_fasta
         File? mt_genes
-        Int tenx_chemistry_version = 3
+        Int tenx_chemistry_version = 2
```
Comment (ekiernan) [inline pipelines/skylab/multiome/Multiome.wdl:21]: @nikellepetrillo Why was the multiome changed to 2? I thought that Multiome chemistry was v3?

Comment (nikellepetrillo) [reply pipelines/skylab/multiome/Multiome.wdl:21]: Thanks for catching this. Changing it back ( i think my local copy of warp is messed up, im going to have to delete and clone again :( )

**Review by ekiernan — APPROVED** [review]
Thank you!

File: pipelines/skylab/multiome/Multiome.changelog.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+# 1.0.1
+2023-07-11 (Date of Last Commit)
+
+* Updated the CountAlignments task in the FeatureCounts.wdl to use a new docker image. This change does not affect the Multiome pipeline.
```
Comment (ekiernan) [inline pipelines/skylab/multiome/Multiome.changelog.md:4]:
```suggestion
* Updated the CountAlignments task in the FeatureCounts.wdl to use a new docker image. This change does not affect the Multiome pipeline

File: pipelines/skylab/smartseq2_single_nucleus_multisample/MultiSampleSmartSeq2SingleNucleus.changelog.md [hunk 3]
```diff
@@ -1,3 +1,8 @@
+# 1.2.25
+2023-07-11 (Date of Last Commit)
+
+* Updated the CountAlignments task in the FeatureCounts.wdl to use a new docker image. This change does not affect the MultiSampleSmartSeq2SingleNucleus pipeline.
```
Comment (ekiernan) [inline pipelines/skylab/smartseq2_single_nucleus_multisample/MultiSampleSmartSeq2SingleNucleus.changelog.md:4]:
```suggestion
* Updated the CountAlignments task in the FeatureCounts.wdl to use a new docker image. This change does not affect the MultiSampleSmartSeq2SingleNucleus pipeline

**Review by kevinpalis — APPROVED** [review]
Looks great!

**Pull request merged**